test/integration/ansible.cfg

```diff
@@ -1,10 +1,10 @@
 [defaults]
 retry_files_enabled = False
 host_key_checking = False
-roles_path = /here/test/integration/targets/
-library = /here/library/modules
-module_utils = /here/library/module_utils
-lookup_plugins = /here/library/plugins/lookup
-action_plugins = /here/library/plugins/action
-terminal_plugins = /here/library/plugins/terminal
+roles_path = targets/
```

Comment:
can you revert this one, we use these ansible.cfg in our containers so the syntax should remain as it was, other than that this looks good

Reply:
Done!